apps/framework-cli/src/cli/commands.rs

```diff
@@ -86,6 +88,17 @@ pub struct GenerateArgs {
 #[derive(Debug, Subcommand)]
 pub enum GenerateCommand {
     Migrations {},
+    Sdk {
+        /// Language of the SDK to be generated
+        #[arg(default_value_t = SupportedLanguages::Typescript, value_enum, short, long)]
+        language: SupportedLanguages,
+        /// Where the SDK files should be written to
+        #[arg(default_value = "./sdk", short, long)]
```

Comment:
do we want to suggest the user to check the SDK into source control, if not should we add this to the default gitignore.

Reply:
yeah I think that's the idea - in connectors for example you want to refer to SDK code directly